src/CodingMilitia.PlayBall.Shared.StartupTasks/DbInitializer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AspNetCore.AsyncInitialization;
+using CodingMilitia.PlayBall.GroupManagement.Data;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+
+namespace CodingMilitia.PlayBall.Shared.StartupTasks
+{
+    public class DbInitializer : IAsyncInitializer
+    {
+        private IWebHost _host;
+        public DbInitializer(IWebHost host)
+        {
+            this._host = host;
+        }
+
+        public async Task InitializeAsync()
+        {
+            using (var scope = _host.Services.CreateScope())
+            {
+                var hostingEnvironment = scope.ServiceProvider.GetRequiredService<IHostingEnvironment>();
+                if (hostingEnvironment.IsDevelopment() || hostingEnvironment.IsEnvironment("DockerDevelopment"))
```

Comment:
I still have some doubts if we should "hardcode" the environments here, or pass them in the constructor, but for now, we can keep them.

Reply:
ill leave a comment on the code as a note